.github/workflows/kyverno-policy.yaml

```diff
@@ -37,14 +37,22 @@ jobs:
       - name: Install Kyvenro
         run: helm install kyverno kyverno/kyverno -n kyverno --create-namespace
 
-      - name: Deploy Workflows
+      - name: Wait for CRDs
         run: |
-          helm install workflows charts/workflows -n workflows --create-namespace
-        
+          kubectl wait --for=condition=Established --timeout=60s crd/sealedsecrets.bitnami.com
+          kubectl wait --for=condition=Established --timeout=60s crd/clusterpolicies.kyverno.io
```

Comment:
`--timeout=60s` for second crd?

Reply:
Yeah, they're from different charts, so the `sealedsecrets` one could arrive before the `kyverno` one